packages/azure-react/src/generators/app-insights-web/files/src/telemetry-provider.tsx__template__

```diff
@@ -0,0 +1,29 @@
+import { FC, ReactNode } from 'react';
+
+import {
+  ReactPlugin,
+  AppInsightsContext,
+} from '@microsoft/applicationinsights-react-js';
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+import appInsightConfig from './app-insights-config';
+
+const reactPlugin = new ReactPlugin();
+const connectionString = process.env.testConnectionString;
```

Comment:
Should it follow env var naming convention and be uppercase?

But actually, I think you meant to interpolate a template variable here?

Reply:
Good catch, I was messing around with it to test it. Will fix it right away.